histomicstk/features/ComputeHaralickFeatures.py

```diff
@@ -0,0 +1,210 @@
+
+        Default: 8 for numeric image, 2 for binary/logical image
+
+    gray_limits : array_like, optional
+        A two-element array specifying the desired input intensity range.
+        Intensity values in the input image will be clipped into this range.
+
+        Default: [0, 1] for binary/logical image, [0, 255] for numeric image
+
+    Returns
+    -------
+    fdata: pandas.DataFrame
+        A pandas dataframe containing the haralick features.
+
+    References
+    ----------
+    .. [1] Haralick, et al. "Textural features for image classification,"
+    IEEE Transactions on Systems, Man, and Cybernatics, vol. 6, pp: 610-621,
+    1973.
+    .. [2] Luis Pedro Coelho. "Mahotas: Open source software for scriptable
+    computer vision," Journal of Open Research Software, vol 1, 2013.
+    """
+
+    # List of feature names
+    feature_list = [
+        'Haralick.ASM.Mean',
+        'Haralick.Contrast.Mean',
+        'Haralick.Correlation.Mean',
+        'Haralick.SumofSquar.Mean',
+        'Haralick.IDM.Mean',
+        'Haralick.SumAverage.Mean',
+        'Haralick.SumVariance.Mean',
+        'Haralick.SumEntropy.Mean',
+        'Haralick.Entropy.Mean',
+        'Haralick.Variance.Mean',
+        'Haralick.DifferenceEntropy.Mean',
+        'Haralick.IMC1.Mean',
+        'Haralick.IMC2.Mean',
+        'Haralick.ASM.Range',
+        'Haralick.Contrast.Range',
+        'Haralick.Correlation.Range',
+        'Haralick.SumofSquar.Range',
+        'Haralick.IDM.Range',
+        'Haralick.SumAverage.Range',
+        'Haralick.SumVariance.Range',
+        'Haralick.SumEntropy.Range',
+        'Haralick.Entropy.Range',
+        'Haralick.Variance.Range',
+        'Haralick.DifferenceEntropy.Range',
+        'Haralick.IMC1.Range',
+        'Haralick.IMC2.Range',
+    ]
+
+    # compute object properties if not provided
+    if rprops is None:
+        rprops = regionprops(im_label)
+
+    # create pandas data frame containing the features for each object
+    numFeatures = len(feature_list)
+    numLabels = len(rprops)
+    fdata = pd.DataFrame(np.zeros((numLabels, numFeatures)),
+                         columns=feature_list)
+
+    # check for consistent shapes between 'I' and 'Label'
+    if im_intensity.shape != im_label.shape:
+        raise ValueError("Inputs 'I' and 'Label' must have same shape")
+
+    # determine if image is grayscale or RGB
+    if len(im_intensity.shape) != 2:  # color image
+        raise ValueError("Inputs 'I' should be a grayscale image")
+
+    for i in range(numLabels):
+        # get bounds of an intensity image
+        minr, minc, maxr, maxc = rprops[i].bbox
+        # grab nucleus mask
+        subImage = im_intensity[minr:maxr+1, minc:maxc+1].astype(np.uint8)
+        # gets GLCM or gray-tone spatial dependence matrix
+        arrayGLCM = graycomatrixext(subImage, offsets=offsets,
+                                    num_levels=num_levels,
+                                    gray_limits=gray_limits)
```

Comment:
I think you have to set `symmetric=True`. See the last paragraph of column-2 in page 3 of [haralick paper](http://haralick.org/journals/TexturalFeatures.pdf)